src/components/MarkdownProvider/components/PropSheets.tsx

```diff
@@ -9,25 +9,17 @@ import React from 'react';
 import { css } from 'styled-components';
 import { ComponentDoc } from 'react-docgen-typescript';
 import { getColor } from '@zendeskgarden/react-theming';
-import { MD, Paragraph } from '@zendeskgarden/react-typography';
+import { Paragraph } from '@zendeskgarden/react-typography';
 import { Table, Head, Body, HeaderRow, HeaderCell, Row, Cell } from '@zendeskgarden/react-tables';
-
-import { IPackage } from './PackageDescription';
 import { StyledH3 } from './Typography';
 
-export const PropSheets: React.FC<{ data: ComponentDoc[]; reactPackage: IPackage }> = ({
-  data,
-  reactPackage
-}) => {
+export const PropSheets: React.FC<{ propSheets: ComponentDoc[] }> = ({ propSheets }) => {
```

Comment:
This component should probably be named `API`. Per below, it would make more sense in context. But that's effort for a separate PR.

```
## Configuration

<Configuration ... />

## API

<API propSheets={props.data.mdx.propsSheets} />
```

Reply:
Capture in 1280